tests/test_utils.py

```diff
@@ -16,15 +16,15 @@ def test_task():
                 result = client.compute(delayed_num).result()
             return result
 
-        @flow(task_runner=DaskTaskRunner)
+        @flow(task_runner=DaskTaskRunner())
```

Comment:
Why these changes?

Reply:
This was based on the type hints, but I see this is incorrect. I've reverted these changes.